src/server/scripts/Spells/spell_item.cpp

```diff
@@ -1059,11 +1059,11 @@ class spell_item_oracle_ablutions : public SpellScriptLoader
                 if (Player* player = caster->ToPlayer())
                 {
                     std::string name = player->GetName();
-                    player->TextEmote("casually flips his Titanium Seal of Dalaran");
+                    player->TextEmote("casually flips his Titanium Seal of Dalaran", 1);
                     if (urand(0,1))
-                        player->TextEmote("finds the coin face down for tails!");
+                        player->TextEmote("finds the coin face down for tails!", 1);
                     else
-                        player->TextEmote("catches the coin heads up!");
+                        player->TextEmote("catches the coin heads up!", 1);
```

Comment:
I would prefer wrapping those two values `1` and `2` into enums and use them accordingly, so it makes the code more readable. For example something like:

```
EMOTE_TYPE_UNIVERSAL = 1
EMOTE_TYPE_FACTION_ONLY = 2
```